app/lib/three_scale/swagger/specification.rb

```diff
@@ -67,6 +69,14 @@ def as_json
         end
       end
 
+      class V31 < V3x
+        JSON_SCHEMA = {'$ref' => 'https://spec.openapis.org/oas/3.1/schema/2022-10-07'}.freeze
+      end
+
+      class V30 < V3x
+        JSON_SCHEMA = {'$ref' => 'https://spec.openapis.org/oas/3.0/schema/2021-09-28'}.freeze
```

Comment:
Updated the previous link because it was broken

Reply:
Was there any consequences of this broken link?